src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -476,15 +487,30 @@ const PIAFormPage = () => {
     if (!pia?.title) {
       invalid = true;
       formId = 'title';
+      setValidationMessages((prevState) => ({
+        ...prevState,
+        piaTitle: 'Error: Please enter a title.',
+      }));
     }
-    if (!pia?.ministry && !invalid) {
```

Comment:
why removing !invalid?

I will let @antsand  - give this a look.

Reply:
I talk to Jordan, She mentioned we should show all the error message at the same time if the user click the submit button

Reply:
There might be an validation abstraction tool to do this in 'Mighty react'.  Is there not? I wrote it in raw JS as it was easier for me to compute.

Reply:
will create a refactor ticket for this